packages/create-plugin/templates/backend-app/pkg/plugin/app.go

```diff
@@ -0,0 +1,39 @@
+package plugin
+
+import (
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/instancemgmt"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/resource/httpadapter"
+)
+
+// Make sure App implements required interfaces. This is important to do
+// since otherwise we will only get a not implemented error response from plugin in
+// runtime. Plugin should not implement all these interfaces - only those which are
+// required for a particular task.
+var (
+	_ backend.CallResourceHandler   = (*App)(nil)
+	_ instancemgmt.InstanceDisposer = (*App)(nil)
+)
```

Comment:
I thought `backend.CheckHealthHandler` was required only to test a datasource connection. Is it also used for other backend plugin types?

Reply:
True required for datasource and more integrated via base class `DataSourceWithBackend`, https://github.com/grafana/grafana/blob/c090de9ed9e3e3865d36a27256a28c8f64179736/packages/grafana-runtime/src/utils/DataSourceWithBackend.ts#L263

But we want apps to implement it as well since we're exposing the API endpoint, e.g. `/api/plugins/<plugin id>/health`.